src/common/components/DefaultTableCellClasses/DefaultTableCellClasses.tsx

```diff
@@ -0,0 +1,2 @@
+import { tableCellClasses as DefaultTableCellClasses } from '@mui/material/TableCell'
```

Comment:
Hier vermutlich auch eher so:
import DefaultTableCellClasses
export DefaultTableCellClasses as TableCellClasses 

wie in den anderen default components

Reply:
Kann nicht als default importiert und named exportiert werden, da es sich hierbei um ein Objekt handelt. Alternativ wäre noch ein direkter export möglich `export { tableCellClasses } from @mui/material/TableCell`, aber dann ergibt die Benennung dieser Datei keinen Sinn mehr. Daher eventuell den direkten Export in die "DefaultTableCell.tsx"-Datei verschieben?

Reply:
Ne dann lassen wir es hier drin würde ich sagen. Hat mich nur gewundert, warum hier nicht die Default sondern direkt importiert wird.